src/release/breaking-changes/window-singleton.md

```diff
@@ -60,7 +62,24 @@ the `PlatformDispatcher` should be accessed via a binding (for example
 `PlatformDispatcher.instance` property. This ensures that the functionality
 of the `PlatformDispatcher` can be properly mocked out in tests.
 
-TODO: Talk about what tests should do.
+### Testing
+
+For tests that accessed the `WidgetTester.binding.window` property to change
+window properties for testing, the following migrations are available:
+
+In tests written with `testWidgets`, two new properties have been added that
+together replace the functionality of `TestWindow`
```

Comment:
nit: this sentence is missing punctuation at the end.

Reply:
Another good catch, added!